test/jsonValidationTest.js

```diff
@@ -5,120 +5,167 @@ const fs = require('fs');
 describe('Validating Cohorts...', () => {
     const cohortPath = './Cohorts';
 
-    it('Verifying .cohort files', function(done){
+    it('Verifying .cohort files', function (done) {
         let failedList = [];
         browseDirectory(cohortPath, (error, results) => {
             results.filter(file => file.substr(-7) === '.cohort')
-                .forEach(file => {                    
-                    validateJsonString(file)
+                .forEach(file => {
+                    validateJsonStringAndGetObject(file)
                 });
-            
-                done();
+
+            done();
         });
     });
 
-    it('Verifying cohort settings files', function(done){
+    it('Verifying cohort settings files', function (done) {
         browseDirectory(cohortPath, (error, results) => {
             if (error) throw error;
             results.filter(file => file.substr(-13) === 'settings.json')
                 .forEach(file => {
-                    validateJsonString(file);
+                    let json = validateJsonStringAndGetObject(file);
+                    validateSettingsForCohort(json, file);
                 });
-            
-                done();
+
+            done();
         });
     });
 
-    it('Verifying cohort category files', function(done){
+    it('Verifying cohort category files', function (done) {
         browseDirectory(cohortPath, (error, results) => {
             if (error) throw error;
             results.filter(file => file.substr(-22) === 'categoryResources.json')
                 .forEach(file => {
-                    validateJsonString(file);
+                    let json = validateJsonStringAndGetObject(file);
```

Comment:
nit: it isn't "json" after this call, it's an actual object.

Reply:
Done